sky/utils/schemas.py

```diff
@@ -106,6 +111,18 @@ def get_resources_schema():
                     'type': 'object',
                     'required': [],
                 }]
+            },
+            'any_of': {
+                'type': 'array',
+                'items': {
+                    'type': 'string',
+                },
```

Comment:
We are taking a list of `resources` schema. How about we define the schema for a single `resources` and have it here?

Reply:
added